plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -363,6 +364,7 @@ function MosaicMenuItem:init()
     -- filepath may be provided as 'file' (history) or 'path' (filechooser)
     -- store it as attribute so we can use it elsewhere
     self.filepath = self.entry.file or self.entry.path
+    -- logger.dbg(self.filepath)
```

Comment:
(Clean these once you don't need them any longer - so we get something clean to review.)